packages/server/graphql/mutations/removePokerTemplateDimension.ts

```diff
@@ -0,0 +1,72 @@
+import {GraphQLID, GraphQLNonNull} from 'graphql'
+import getRethink from '../../database/rethinkDriver'
+import {getUserId, isTeamMember} from '../../utils/authorization'
+import publish from '../../utils/publish'
+import standardError from '../../utils/standardError'
+import RemovePokerTemplateDimensionPayload from '../types/RemovePokerTemplateDimensionPayload'
+import {SubscriptionChannel} from 'parabol-client/types/constEnums'
+
+const removePokerTemplateDimension = {
+  description: 'Remove a dimension from a template',
+  type: RemovePokerTemplateDimensionPayload,
+  args: {
+    dimensionId: {
+      type: new GraphQLNonNull(GraphQLID)
+    }
+  },
+  async resolve(_source, {dimensionId}, {authToken, dataLoader, socketId: mutatorId}) {
+    const r = await getRethink()
+    const now = new Date()
+    const operationId = dataLoader.share()
+    const subOptions = {operationId, mutatorId}
+    const dimension = await r
+      .table('TemplateDimension')
+      .get(dimensionId)
+      .run()
+    const viewerId = getUserId(authToken)
+
+    // AUTH
+    if (!dimension || !isTeamMember(authToken, dimension.teamId) || !dimension.isActive) {
+      return standardError(new Error('Team not found'), {userId: viewerId})
+    }
+
+    // VALIDATION
+    const {teamId, templateId} = dimension
+    const dimensionCount = await r
+      .table('TemplateDimension')
+      .getAll(teamId, {index: 'teamId'})
+      .filter({
+        isActive: true,
```

Comment:
@nickoferrall came up with a solid pattern which we may want to implement here.

the problem is a dimension might get removed while a meeting is in progress or after a meeting has ended.
if i re-visit an ended meeting, i should still be able to see that removed dimension.

this can be accomplished by replacing `isActive` with a `removedAt: Date` field. 
The logic is pretty simple: https://github.com/ParabolInc/parabol/blob/7c207c7651c38003ef1e3a88c05ce2314b0a4001/packages/server/utils/isRecordActiveForMeeting.ts